src/pages/lib-v1/auth/switch-auth/q/platform/[platform].mdx

```diff
@@ -1,4 +1,58 @@
 export const meta = {
   title: `Switching authentication flows`,
-  description: `Learn more about how to switch between different auth flows in Amplify.`,
-};
\ No newline at end of file
+  description: `Learn more about how to switch between different auth flows in Amplify.`
+};
+
```

Comment:
I think all content on this page should be wrapped in `<InlineFilter filters={["js"]}>`